descwl_coadd_task/tests/test_warps.py

```diff
@@ -25,8 +25,10 @@ def setUpClass(cls):
         # Docstring inherited.
         # This method contains the setup that needs to be done only once for
         # all the tests.
-        cls.ny = 4000
-        cls.nx = 4072
+        # cls.ny = 4000
+        # cls.nx = 4072
+        cls.ny = 150
+        cls.nx = 100
```

Comment:
What is being commented out here and why?

Reply:
I made the images smaller to speed up tests.  If there is a reason to use full sized we could undo this.

Reply:
The reason for the large size was to have the scenario where the warp has images from multiple detectors, and chip gaps in between. It doesn't have to be full-size, but large enough to include that scenario. I can find an optimal size, but larger than 150x100 when adding coadd tests.